kimchi/src/circuits/lookup/tables/mod.rs

```diff
@@ -1,45 +1,64 @@
-use ark_ff::{FftField, One, Zero};
+use ark_ff::{Field, One, Zero};
 use poly_commitment::PolyComm;
 use serde::{Deserialize, Serialize};
+use thiserror::Error;
 
 pub mod range_check;
 pub mod xor;
 
-//~ spec:startcode
```

Comment:
Why is it moved down? If no reason, it is adding noise to the PR.

Reply:
It makes more sense in terms of file structure:
1. First we define `LookupTable`
2. Then there's code on fixed tables (xor & range). Note that in the original this code is split in two, `gate_table` and `impl GateLookupTable` were somewhat astray from everything else.
3. Then there's generic code on merging tables.

Ideally fixed lookup tables should be in a separate module, but I thought it might create too many conflicts with the customizable kimchi effort, where it'll be probably done later anyway.